sledilnik_mobile_app/lib/ui/widgets/info_box.dart

```diff
@@ -33,7 +32,7 @@ class InfoBox extends StatelessWidget {
     final trendColor = colorScheme.getTrendColor(trendType);
     return Container(
         padding: const EdgeInsets.symmetric(horizontal: 8.0, vertical: 8.0),
-        decoration: BoxDecoration(color: Colors.yellow[50]),
+        decoration: BoxDecoration(color: Color(0xfffffbe8)),
```

Comment:
Can you move this color to CustomColorScheme and assign a name to it?

Reply:
Taken care of. I expect we'll need more changes later on, as the app evolves. For now I named it primaryYellow.

Reply:
Can you name this guy as well? `decoration: BoxDecoration(color: Color(0xfffffbe8)),`